docker-compose.yaml

```diff
@@ -0,0 +1,40 @@
+version: '3'
+services:
+  
+  spire-server:
+    image: gcr.io/spiffe-io/spire-server:0.12.0
+    container_name: test-infra_spire-server_1
+    hostname: spire-server
+    volumes:
+        - ./test-infra/spire/server:/opt/spire/conf/server
+        - /tmp/spire/sockets:/run/spire/sockets
+    command: ["-config", "/opt/spire/conf/server/server.conf"]
+
+  spire-agent:
+    pid: "host"
+    container_name: test-infra_spire-agent_1
+    privileged: true
+    image: gcr.io/spiffe-io/spire-agent:0.12.0
+    depends_on: ["spire-server"]
+    hostname: spire-agent
+    volumes:
+        - ./test-infra/spire/agent:/opt/spire/conf/agent
+        - /tmp/spire/sockets:/run/spire/sockets
+        - /proc:/proc
+        - ./test-infra/certs:/certs
+    command: ["-config", "/opt/spire/conf/agent/agent.conf"]
+
+  intoto-runner:
+    container_name: intoto-runner
+    image: intoto-run:latest
+    privileged: true
+    entrypoint: sleep 1000
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    volumes:
+        - /tmp/spire/sockets:/run/spire/sockets
+        - ./test/tmp:/test/tmp
+        - ./certs:/certs
+    depends_on:
+      - spire-agent
```

Comment:
I'm leaning towards moving this into a test directory so it doesn't get used as a production file. The version should also probably be bumped down to 2, depends_on was removed in version 3.

Reply:
Moved to test-infra directory and changed version to 2. I didnt realize they removed that depends_on on version 3.